bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -238,7 +238,6 @@ myVar = 'Hello mutation!'; // When the timer runs, it will log "Hello mutation!"
 
 If you need to pass some variables to the timer but avoid that they can get mutated, pass those variables as parameters to `setTimeout`/`setInterval` or `createTimer`:
 
-
```

Comment:
This needs to be fixed in the openhab-js repository, otherwise will get overwritten.

Reply:
Reverted

Reply:
I think you need to rebase? It still shows as difference.

Reply:
The change was there. Seems there was another one on line 703. Reverted that too.